composer/trainer/trainer.py

```diff
@@ -1675,6 +1675,7 @@ def fit(
 
         # Max Duration
         if duration is not None:
+            log.info(str(duration))
```

Comment:
Is this log intentional? If so it should probably provide some more info about what it is logging